### DISCUSSION TITLE
Support for Wedo 2.0

### DISCUSSION THREAD

**Discussion by wolli112** [pybricks/support#658] [Ideas]
Is it already possible to use PyBricks on a Wedo 2.0 component?

Like this one?
![IMG_9311](https://user-images.githubusercontent.com/103441220/171044198-36d9b411-bc21-4e5a-823a-0e41fb13405e.JPG)

Lego Education 45300 Wedo 2.0

**Comment by dlech** [comment]
The WeDo 2.0 hub does not have updateable firmware, so it is not possible to run Pybricks on the hub. There are other libraries that can be used to remotely control the hub instead.

**Comment by carlossolis27** [comment]
Hi

May i ask you the name of that book ? it looks interesting for continue learning sorry to bother you

**Reply by dlech** [reply]
This is a javascript library that can do the job: https://github.com/nathankellenicki/node-poweredup

**Reply by laurensvalk** [reply]
@carlossolis27 - It looks like this is the reference sheet in that picture: https://perso.limsi.fr/pointal/_media/python:cours:mementopython3-english.pdf
